prime-router/src/test/kotlin/fhirengine/engine/FhirConverterTests.kt

```diff
@@ -340,4 +361,110 @@ class FhirConverterTests {
             actionHistory.trackCreatedReport(any(), any(), blobInfo = any())
         }
     }
+
+    @Test
+    fun `test condition code stamping`() {
```

Comment:
Nit: maybe one more test here or as a DB test that checks that the case when one condition can be mapped, but another can't be?

Reply:
Added another test case to verify